interpreter/mesh/src/main/scala/io/buoyant/interpreter/MeshInterpreterInitializer.scala

```diff
@@ -43,7 +43,7 @@ case class Retry(
 case class MeshInterpreterConfig(
   dst: Option[Path],
   root: Option[Path],
-  tls: Option[TlsClientConfig],
+  tls: Option[ClientTlsConfig],
```

Comment:
Isn't this a breaking change?  I think we'd like to standardize on TlsClientConfig as the proper way to specify client tls configuration.  It's somewhat unfortunate that the namerd thrift interpreter uses a different format but we're stuck with it until a breaking release I think.